bps/templates/bps_flat_g.yaml

```diff
@@ -0,0 +1,11 @@
+pipelineYaml: "${CP_PIPE_DIR}/pipelines/${INSTRUMENT}/cpFlat.yaml"
+
+project: "${TICKET}"
+campaign: "${TICKET}"
+
+payload:
+  payloadName: "${INSTRUMENT}_${TICKET}"
```

Comment:
Does the `payloadName` need to be unique?  Will it complain if you launch flats in all filters simultaneously?  I supposed this could apply to any of the steps, but flats feel more parallelizable than the others.